Patch created by brosx "diffPackage" tool

usage:
    diffPackage <tarName> <pkgName> <outName> <patchName> <files>

actual:
    diffPackage "acl-2.2.52.src.tar.gz" "acl" "host-acl" "care-for-gettext-autoconf-param" "\
        po/Makefile \
    ";


===============================================================================
--- a/po/Makefile	2013-04-30 17:20:43.000000000 +0200
+++ b/po/Makefile	2018-01-03 18:09:08.000000000 +0100
@@ -19,6 +19,8 @@
 TOPDIR = ..
 include $(TOPDIR)/include/builddefs
 
+ifeq ($(ENABLE_GETTEXT), yes)
+
 POTHEAD = $(PKG_NAME).pot
 LINGUAS ?= de es fr gl pl sv
 LSRCFILES = $(LINGUAS:%=%.po) $(POTHEAD)
@@ -38,5 +40,9 @@
 
 install: default
 	$(INSTALL_LINGUAS)
+else
+include $(BUILDRULES)
+install:
+endif
 
 install-dev install-lib:
